### ISSUE TITLE
Figure out how to run mypy properly

### ISSUE THREAD

**Issue by jtojnar** [jtojnar/what-changed#5]
Currently, I need to pass libversion types manually and it still fails due to bs [not providing type annotations](https://bugs.launchpad.net/beautifulsoup/+bug/1843791):

```
$ env MYPYPATH=/nix/store/dlfbmcjpdspq8daw8rx64br1cz4cdycv-python3.7-libversion-1.1.4/lib/python3.7/site-packages/ nix-shell -p python3 -p python3Packages.mypy -p python3.pkgs.libversion -p python3.pkgs.beautifulsoup4 -p python3.pkgs.requests --run 'python -m mypy changelogs.py'
changelogs.py:4: error: No library stub file for module 'bs4'
changelogs.py:4: note: (Stub files are from https://github.com/python/typeshed)
```

**Comment by jtojnar** [comment]
Now that we got rid of BeautifulSoup in 1825700304daaeb8f4f821b82465d19c5aebd856, there is only the libversion issue remaining:

```ShellSession
$ nix-shell -p python3 -p python3Packages.mypy -p python3.pkgs.libversion -p python3.pkgs.requests --run 'python -m mypy changelogs.py'
changelogs.py:4: error: Cannot find module named 'libversion'
changelogs.py:4: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
```

**Comment by jtojnar** [comment]
Fixed in 88571fe3a925b92d8073590bfd3c8b26a2af6da4 using `--ignore-missing-imports`
